### ISSUE TITLE
Add explanation for CW41, CW42, etc. date labels on yearly/all-time graphs

### ISSUE THREAD

**Issue by stevennyman** [ihucos/counter.dev#106]
Apparently they stand for calendar week 41, calendar week 42, etc., but it wasn't fully clear what the labels meant before I looked them up. Perhaps it would make sense to show a non-abbreviated version of these labels and/or the date ranges they represent on hover?

**Comment by ihucos** [comment]
Makes sense. Any more concrete ideas how to display that? Hmm writing "Calendar Week" would overflow the graph.

**Comment by stevennyman** [comment]
Maybe as a tooltip when you hover on the bottom label or something?

**Comment by ihucos** [comment]
Yes, that might be a good idea, let's see if Chart.js supports it... thx
